### ISSUE TITLE
Time-based DKP is always awarded to Standby users even when they are offline

### ISSUE THREAD

**Issue by f90** [Roeshambo/MonolithDKP#309]
When Auto-DKP Award system is set to "Only online players", I would expect that time-based interval DKP are only assigned to those that are online with their characters both in the raid and on the extra standby list. However, it seems that standbys ALWAYS get the time-based DKP even if offline, while raiders DONT get the DKP when they are offline. 

Standbys should also be checked for whether they are online or not.

**Comment by kohoward** [comment]
Does it mean if they park their main outside and switch to an alt, they would get nothing? What if they stay on voice chat?

This seems to limit functionality.

If you really want an automatic way to handle this, then there needs to be 2 separate check online functions for in-raid and standbys.

**Comment by f90** [comment]
Yeah would be good if you have the choice separately for raiders and standby whether to check for online and same-zone conditions
